website/vercel.json

```diff
@@ -2,21 +2,23 @@
   "headers": [
     {
       "source": "/(.*)",
-      "headers" : [
+      "headers": [
         {
-          "key" : "X-Robots-Tag",
-          "value" : "all"
+          "key": "X-Robots-Tag",
+          "value": "all"
         }
       ]
     },
     {
       "source": "/testing(.*)",
-      "headers" : [
+      "headers": [
         {
-          "key" : "X-Robots-Tag",
-          "value" : "none"
+          "key": "X-Robots-Tag",
+          "value": "none"
         }
       ]
     }
-  ]
+  ],
+  "framework": "ember",
+  "rewrites": [{ "source": "/(.*)", "destination": "/_empty.html" }]
```

Comment:
It seemed odd to me at first too @alex-ju -- but here's how I understand it to work (maybe it will help)

1. Prember provides all of the URLs that will exist when we build, so they are in the Output Directory
2. this is saying "if a user tries to go to any page that doesn't exist in that Output Directory, route them _here_.